packages/data-context/test/unit/sources/migration/codegen.spec.ts

```diff
@@ -58,6 +58,34 @@ describe('cypress.config.js generation', () => {
       },
     }
 
+    const generatedConfig = await createConfigString(config, {
+      hasE2ESpec: true,
+      hasComponentTesting: true,
+      hasPluginsFile: true,
+      projectRoot,
+      hasTypescript: false,
+    })
+
+    snapshot(generatedConfig)
+  })
+
+  it('should create only a component entry when no e2e specs are detected', async () => {
```

Comment:
@lmiller1990 - I added this test and left the one below it, testing the case where only one testing type is detected. The snapshots for these two test display only `component` and only `e2e` blocks respectively.

Reply:
Thanks!